### PR TITLE
Copernicus Open Acess Hub Url changed

### LINKED ISSUE #142: Update OA hub URL
[Copernicus Open Access Hub changed the URL](https://scihub.copernicus.eu/news/News00868) and we are currently getting `401 Unauthorized`.

### PR DESCRIPTION
**Pull request by shiwakotisurendra**
closes #142

### CODE REVIEW

File: ukis_pysat/data.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def __init__(self, datahub, catalog=None, url=None):
             self.api = sentinelsat.SentinelAPI(
                 self.user,
                 self.pw,
-                "https://scihub.copernicus.eu/apihub",
+                "https://apihub.copernicus.eu/apihub",
```
Comment (shiwakotisurendra) [inline ukis_pysat/data.py:90]: i found the error as well and changed the OA url as you suggested.

File: tests/test_data.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def test_query_metadata_earth_explorer(self, m):
     @requests_mock.Mocker(real_http=True)
     def test_query_metadata_srcid_scihub(self, m):
         m.post(
-            "https://scihub.copernicus.eu/apihub/search?format=json&rows=100&start=0",
+            "https://apihub.copernicus.eu/apihub/search?format=json&rows=100&start=0",
```
Comment (shiwakotisurendra) [inline tests/test_data.py:134]: Error found and resolved according to the suggestions. Test is also working fine now.

File: ukis_pysat/data.py (hunk 1 above)
Comment (fwfichtner) [reply ukis_pysat/data.py:90]: Good!

File: tests/test_data.py (hunk 2 above)
Comment (fwfichtner) [reply tests/test_data.py:134]: When these are all, it is fine. Thanks!

**Review by fwfichtner — APPROVED** [review]

File: tests/test_data.py [hunk 3]
```diff
@@ -131,7 +131,7 @@ def test_query_metadata_earth_explorer(self, m):
...
...entname","content":"Synthetic Aperture Radar (C-band)"},{"name":"instrumentshortname","content":"SAR-C SAR"},{"name":"filename","content":"S1A_IW_SLC__1SDV_20200224T052528_20200224T052555_031390_039CF2_BEA6.SAFE"},{"name":"format","content":"SAFE"},{"name":"productclass","content":"S"},{"name":"polarisationmode","content":"VV VH"},{"name":"acquisitiontype","content":"NOMINAL"},{"name":"status","content":"ARCHIVED"},{"name":"size","content":"7.65 GB"},{"name":"gmlfootprint","content":"<gml:Polygon srsName=\\"http://www.opengis.net/gml/srs/epsg.xml#4326\\" xmlns:gml=\\"http://www.opengis.net/gml\\">\\n   <gml:outerBoundaryIs>\\n      <gml:LinearRing>\\n         <gml:coordinates>50.907326,13.664250 51.312340,10.003944 52.933346,10.404160 52.525398,14.201497 50.907326,13.664250</gml:coordinates>\\n      </gml:LinearRing>\\n   </gml:outerBoundaryIs>\\n</gml:Polygon>"},{"name":"footprint","content":"MULTIPOLYGON (((13.66425 50.907326, 14.201497 52.525398, 10.40416 52.933346, 10.003944 51.31234, 13.66425 50.907326)))"},{"name":"identifier","content":"S1A_IW_SLC__1SDV_20200224T052528_20200224T052555_031390_039CF2_BEA6"},{"name":"uuid","content":"8a611d5b-f9d9-437e-9f55-eca18cf79fd4"}]}}}',
```
Comment (fwfichtner) [inline tests/test_data.py:135]: Did you re-record the content?

Comment (shiwakotisurendra) [reply tests/test_data.py:135]: no i didnt

**Review by fwfichtner — APPROVED** [review]

**Pull request merged**